src/pageContext/withPageContext.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { IntlProvider, addLocaleData } from "react-intl";
+import csData from "react-intl/locale-data/cs";
+import enData from "react-intl/locale-data/en";
+
+import PageContext from "./PageContext";
+
+import { translations } from "../i18n";
+
+import "../styles/common.sass";
```

Comment:
Functional HOC having styles imported? I see that first time, is it relevant? Why it can't be imported in some layout component?

Reply:
because I'm a lazy bitch? :D ok, legit, i'll do it